### ISSUE TITLE
Update README.md with Netcat as a requirement

### ISSUE THREAD

**Issue by wattry** [OSC/bc_example_jupyter#7]
The script that is generated by the example uses Netcat , this will cause failures.

 The rstudio docs have contain nc as a prerequisite.

**Comment by ericfranz** [comment]
Would it be preferable for us to modify the code to not use netcat, or try netstat if netcat is not available i.e. [your suggestion](https://discourse.osc.edu/t/jupyter-notebook-connect-to-jupyter-button-not-rendering/225/2):

```bash
port_used () {
  local port="{1#*:}" sleep 2 netstat -nap 2>/dev/null | grep -q "0.0.0.0:{port} "
}
```

**Comment by wattry** [comment]
Well, that would preferable for us, our ops team won't permit applications that may make our systems more vulnerable. 

It's really your call, I just figured it would be easier for you to update the README then to update the script. It took a few hours to debug this because no error is output when the nc call fails.

**Comment by MorganRodgers** [comment]
I agree that if there are problems with asking sites to install netcat then we should do something else.

But, as for the correct fix for this issue, netcat is listed as a dependency in the setting up interactive apps section of the [OnDemand documentation for version 1.5 / master](https://osc.github.io/ood-documentation/master/app-development/interactive/setup/software-requirements.html). Example interactive apps need to link to that documentation.
